src/odemis/driver/test/actuator_test.py

```diff
@@ -548,6 +548,49 @@ def test_metadata(self):
 
         testing.assert_pos_almost_equal(stage.speed.value, {"x": 10e-6, "y": 20e-6})
 
+    def test_get_axes_range(self):
+        """
+        Test the range calculation in ConvertStage
+        """
+        # Test based on given range of Zeiss stage
+        dependency = simulated.Stage("stage", "test", axes=["y", "m"],
+                                     ranges={"y": [0.e-3, 100.e-3], "m": [0.e-3, 13.e-3]})
```

Comment:
I understand it's based on a real world example, however for the test case, we don't need all this extra info. You can keep it abstract and simple. So you can drop the comment that it's based on Zeiss, or that it's a "Meteor stage". Also, can you simplify and name the axes more simpler values, like a & b, or u & v?